css-editor/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import InputsBox from './InputsBox.js'
+import Background from './Background'
+import './App.css';
+
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      backGroundColor: null,
+      headerTitle: null,
+      headerColor: null,
+      mainColor: null,
+      mainText: '',
+      submitted: false
+    }
+  }
+  
+  onChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+   
+  };
+
+  onSubmit = e => {
+    this.setState({
+      submitted: true
+    })
+    console.log(this.state.backGroundColor)
```

Comment:
if possible remove any console.log's before production, but since its a hackathon I'm not too sure if were following the same conventions